### PR TITLE
Sentencepiece recipe

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

File: conda-recipes/sentencepiece-feedstock/recipe/tensorflow-datasets-test [hunk 1]
```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```
Comment (jayfurmanek) [inline conda-recipes/sentencepiece-feedstock/recipe/tensorflow-datasets-test]: Probably dont need this file.

File: conda-recipes/sentencepiece-feedstock/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+requirements:
+  build:
+    - cmake >=3.1
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - tensorflow
```
Comment (jayfurmanek) [inline conda-recipes/sentencepiece-feedstock/recipe/meta.yaml]: Maybe we should have tensorflow-base here?

File: conda-recipes/sentencepiece-feedstock/recipe/tensorflow-datasets-test (hunk 1 above)
Comment (npanpaliya) [reply conda-recipes/sentencepiece-feedstock/recipe/tensorflow-datasets-test]: Ohh, yeah. Copy paste error :)

**Comment by npanpaliya** [comment]
Yeah, build wise this is okay but import is failing for built package which I'm looking into currently. On our RHEL containers, it had worked but here it is failing after installing conda package.
`>>> import sentencepiece
Traceback (most recent call last):
  File "/opt/conda/envs/testsp/lib/python3.6/site-packages/sentencepiece.py", line 18, in swig_import_helper
    return importlib.import_module(mname)
  File "/opt/conda/envs/testsp/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 658, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: /opt/conda/envs/testsp/lib/python3.6/site-packages/_sentencepiece.cpython-36m-powerpc64le-linux-gnu.so: undefined symbol: _ZNK13sentencepiece4util6Status8ToStringB5cxx11Ev

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/envs/testsp/lib/python3.6/site-packages/sentencepiece.py", line 21, in <module>
    _sentencepiece = swig_import_helper()
  File "/opt/conda/envs/testsp/lib/python3.6/site-packages/sentencepiece.py", line 20, in swig_import_helper
    return importlib.import_module('_sentencepiece')
  File "/opt/conda/envs/testsp/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: /opt/conda/envs/testsp/lib/python3.6/site-packages/_sentencepiece.cpython-36m-powerpc64le-linux-gnu.so: undefined symbol: _ZNK13sentencepiece4util6Status8ToStringB5cxx11Ev`

**Pull request merged**